pkg/plugins/manager/testdata/valid-model-datasource/composable_dataquery.cue

```diff
@@ -0,0 +1,13 @@
+package grafanaplugin
```

Comment:
This is more of a personal preference but not a big fan of the `composable_` notation. I would prefer to use dataquery.cue in this case.